<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Turkish" country="Turkey"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Aramaya Basla!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Durdur!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <property name="title">
     <string>Bilgi Ekrani</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <property name="title">
     <string>Iletisim Ayarlari</string>
    </property>
    <widget class="QCheckBox" name="checkBox_sound">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
      </font>
     </property>
     <property name="text">
      <string>Bilet bulunursa ses cal</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <property name="title">
     <string>Telegram Ayarları</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Microsoft New Tai Lue</family>
         </font>
        </property>
        <property name="text">
         <string>Bot Token:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Microsoft New Tai Lue</family>
         </font>
        </property>
        <property name="text">
         <string>Chat ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="botToken_text">
        <property name="font">
         <font>
          <family>Microsoft New Tai Lue</family>
         </font>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="chatId_text">
        <property name="font">
         <font>
          <family>Microsoft New Tai Lue</family>
         </font>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <property name="title">
     <string>Aranilan Tren Bilgileri</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Microsoft New Tai Lue</family>
           </font>
          </property>
          <property name="text">
           <string>Arama Sıklığını seçiniz: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Microsoft New Tai Lue</family>
           </font>
          </property>
          <property name="text">
           <string>1 dakika</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_period">
        <property name="font">
         <font>
          <family>Microsoft New Tai Lue</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>102</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_nereye">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="text">
        <string>Nereye:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_nereden">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="text">
        <string>Tarih / Saat:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="text">
        <string>Nereden:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionHakk_nda"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHakk_nda">
   <property name="text">
    <string>Hakkında</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
